1.9/administration/id-and-access-mgt/add-user-script.md

```diff
@@ -0,0 +1,22 @@
+---
+post_title: Adding Users Manually
+menu_order: 4
+---
+
+You can add users to your DC/OS cluster from a terminal by using the `dcos_add_user.py` script. This script is included by default with your DC/OS installation.
+
+**Prerequisites:**
+
+- DC/OS is [installed](/docs/1.9/administration/installing/)
+
+1.  [SSH](/docs/1.9/administration/access-node/sshcluster/) to a master node and run this command, where `<email>` is the user's email:
+
+    ```bash
+    $ dcos-shell dcos_add_user.py <email>
```

Comment:
`sudo dcos-shell dcos_add_user.py <email>`

We should also double-check this works.